modules/internal/ArrayViewRankChange.chpl

```diff
@@ -280,15 +571,15 @@ module ArrayViewRankChange {
       // see what subset of target locales the rank-change slice hit vs.
       // not.
       //
-      compilerWarning("Calls to .targetLocales() on rank-change slices may currently return a superset of the locales targeted.");
+      //      compilerWarning("Calls to .targetLocales() on rank-change slices may currently return a superset of the locales targeted.");
       return arr.dsiTargetLocales();
     }
 
     proc dsiHasSingleLocalSubdomain() param
-      return privDom.dsiHasSingleLocalSubdomain();
+      return privDom.updom.dsiHasSingleLocalSubdomain();
```

Comment:
Hmm, good catch.  This looks like a bug to me and I'm guessing reveals another routine that isn't tested by the test system at present (but which my userAPI test would catch.  Finishing that test was going to be code freeze week work).  Let me write a test of this specific case before merging.